src/fonduer/supervision/labeler.py

```diff
@@ -82,13 +89,7 @@ def apply(
             # Call apply on the specified docs for all splits
             split = ALL_SPLITS
             super(Labeler, self).apply(
-                docs,
-                split=split,
-                train=train,
-                update=update,
```

Comment:
why remove update?

Reply:
Now that we are using a upsert for everything, update doesn't really serve a purpose.